### DISCUSSION TITLE
Variables Inside max/minValue

### DISCUSSION THREAD

**Discussion by AKA-Mae** [mProjectsCode/obsidian-meta-bind-plugin#283] [Q&A] [answered]
Is it at all possible to use an existing variable inside the max/minValue fields?
For example, to make a health bar using the progress bar:
````
```meta-bind
INPUT[progressBar(
minValue(0),
maxValue({maxHP})
):HP]
```
````
I have the value maxHP defined as a number from a prior field, however I am getting an error that the value of argument `maxValue` must be of type number
I have tried several different ways so far, to no avail, such as `maxValue([{maxHP}])`, `maxValue(maxHP)`, and `maxValue[{maxHP}]` to no avail

**Comment by mProjectsCode** [comment, marked as answer]
No, it is not without using JS. You can probably adapt [this example](<https://www.moritzjung.dev/obsidian-meta-bind-plugin-docs/guides/advancedusecases/#dynamic-options-for-select-inputs>) from the docs though.

**Reply by AKA-Mae** [reply]
Good to know, thanks for the help!
